include/exadg/operators/integrator_flags.h

```diff
@@ -22,54 +22,66 @@
 #ifndef INCLUDE_EXADG_OPERATORS_INTEGRATOR_FLAGS_H_
 #define INCLUDE_EXADG_OPERATORS_INTEGRATOR_FLAGS_H_
 
+#include <deal.II/matrix_free/evaluation_flags.h>
+
 namespace ExaDG
 {
 struct CellFlags
 {
-  CellFlags(bool const value = false, bool const gradient = false, const bool hessian = false)
-    : value(value), gradient(gradient), hessian(hessian){};
+  explicit CellFlags(
+    dealii::EvaluationFlags::EvaluationFlags const flags_in = dealii::EvaluationFlags::nothing)
+    : flags(flags_in){};
 
   CellFlags
   operator||(CellFlags const & other)
```

Comment:
Since we changed the logic from boolean to bitwise operations (which comes from deal.II), it would be consequent to call this operator `|` instead of `||` in my opinion.

Reply:
I agree

Reply:
Resolved in dd1ef5e5ad50ad77ac7909ef32267ccf8019778d